packages/ddp-server/writefence.js

```diff
@@ -59,6 +60,15 @@ _.extend(DDPServer._WriteFence.prototype, {
     self._maybeFire();
   },
 
+  // Register a function to be called before firing the fence.
```

Comment:
Be more explicit: "It's OK for this function to call `beginWrite`, in which case the fence won't fire until those writes are done as a well.  However, the function will only be called once."

(That's OK, right?  We're not going to add any more "real" writes while the "catch-up" write is catching up, right?)